edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.Amqp/linkhandlers/LinkHandler.cs

```diff
@@ -41,10 +41,12 @@ public abstract class LinkHandler : ILinkHandler
             }
 
             string modelId = null;
-            if (this.Link.Settings?.Properties?.TryGetValue(IotHubAmqpProperty.ModelId, out modelId) ?? false)
-            {
-                this.ModelId = Option.Maybe(modelId);
-            }
+            this.Link.Settings?.Properties?.TryGetValue(IotHubAmqpProperty.ModelId, out modelId);
+            this.ModelId = Option.Maybe(modelId);
+
+            string authChain = null;
```

Comment:
nit: can you combine this line and the next?

Reply:
aren't they already combined?  or are you talking about combining both model ID and auth-chain sections?